l10n_eu_oss_oca/wizard/l10n_eu_oss_wizard.py

```diff
@@ -0,0 +1,237 @@
+                self.env._(
+                    "The Europe country group cannot be found. "
+                    "Please update the base module."
+                )
+            )
+        return eu_group
+
+    def _default_fiscal_position_id(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        return self.env["account.fiscal.position"].search(
+            [
+                ("company_id", "=", user.company_id.id),
+                ("vat_required", "=", True),
+                ("country_group_id", "=", eu_country_group.id),
+            ],
+            limit=1,
+        )
+
+    def _default_done_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        return (
+            eu_country_group.country_ids
+            - self._default_todo_country_ids()
+            - user.company_id.country_id
+        )
+
+    def _default_todo_country_ids(self):
+        user = self.env.user
+        eu_country_group = self._get_eu_res_country_group()
+        eu_fiscal = self.env["account.fiscal.position"].search(
+            [
+                ("country_id", "in", eu_country_group.country_ids.ids),
+                ("vat_required", "=", False),
+                ("auto_apply", "=", True),
+                ("company_id", "=", user.company_id.id),
+                ("fiscal_position_type", "=", "b2c"),
+            ]
+        )
+        return (
+            eu_country_group.country_ids
+            - eu_fiscal.mapped("country_id")
+            - user.company_id.country_id
+        )
+
+    company_id = fields.Many2one(
+        "res.company", string="Company", required=True, default=_get_default_company_id
+    )
+    done_country_ids = fields.Many2many(
+        "res.country",
+        "l10n_eu_oss_country_rel_done",
+        default=_default_done_country_ids,
+        string="Already Supported",
+    )
+    todo_country_ids = fields.Many2many(
+        "res.country",
+        "l10n_eu_oss_country_rel_todo",
+        default=_default_todo_country_ids,
+        string="EU Customers From",
+        required=True,
+    )
+    price_include_tax = fields.Boolean(default=False)
+    general_tax = fields.Many2one(comodel_name="account.tax", required=True)
+    reduced_tax = fields.Many2one(
+        comodel_name="account.tax",
+    )
+    superreduced_tax = fields.Many2one(
+        comodel_name="account.tax",
+        string="Super Reduced Tax",
+    )
+    second_superreduced_tax = fields.Many2one(
+        comodel_name="account.tax", string="Second Super Reduced Tax"
+    )
+
+    def _prepare_tax_group_vals(self, rate):
+        return {
+            "name": self.env._("OSS VAT %s%%") % rate,
+            "country_id": self.company_id.account_fiscal_country_id.id,
+        }
+
+    def _prepare_repartition_line_vals(self, original_rep_lines):
+        return [
+            (
+                0,
+                0,
+                {
```

Comment:
Please use [Command.create](https://github.com/odoo/odoo/blob/de75967b81e29464b5c8726066cfb45ad2842a9f/odoo/fields.py#L4238); it’s more readable
```suggestion
            Command.create(
                {
```

Reply:
This too :)